### PR TITLE
Adds The "Songs of Praise" Sect

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Values have not been tested well given that I cannot replicate someone playing the song in the middle of a crowd.

## About The Pull Request

Adds the "Songs of Praise" Sect, a sect dedicated to playing the FINEST of tunes.

They lose their bible healing, but can perform rites which affect the next song they play.

To clarify, this is a song YOU input. Anything from moonlight sonata to fighting gold, you decide! Get those guitars and pianos out!

Currently they have 4 rites:
Soothing Hymn - Sings a soothing hymn, healing those around you. Holy roles get higher heals.
Sorrow Song - Sings a very sad song, causing pain to those around you. Holy roles take less damage.
Lullaby - Sings a soft lullaby, makes those around them very drowsy, eventually causing them to sleepy. Holy Roles are immune.
Evangelical Hymn - Gains favor for your sect for each non-convert listening.

## Why It's Good For The Game

Adds a new way for chaplains to play with the sect mechanic.

## Changelog
:cl: Cobby
add: Be the idol you always wanted to be with the newest "Songs of Praise" sect! Woo (and heal!) the crowd with Clair De Lune, or hurt them with Caramelldansen! 
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
I don't like these ideas, chaplain is the last thing that needs more bloat and *magic* healing.

Sects should be nothing more than flavour and RP potential, this stuff is just rediculous.

File: code/modules/religion/rites/musician_rites.dm [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+/datum/religion_rites/song_tuner/evangelism/song_effect(atom/A, datum/song/S)
+	if(!S || !GLOB.religious_sect)
+		return
+	for(var/i in S.hearing_mobs)
+		if(i == A)
+			continue
+		if(!isliving(i))
+			continue //stinky ghosts
+		var/mob/living/L = i
+		if(L.mind.holy_role)
```
Comment (ATH1909) [inline code/modules/religion/rites/musician_rites.dm]: you should probably add a mind check somewhere here, so you can't just farm this with NPC monkeys or bees or mice or whatever

File: code/modules/religion/rites/musician_rites.dm [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	if(!S)
+		return
+	for(var/i in S.hearing_mobs)
+		if(i == A)
+			continue
+		if(!isliving(i))
+			continue
+		var/mob/living/L = i
+		var/pain_juice = 0.5
+		if(L.mind.holy_role)
```
Comment (ATH1909) [inline code/modules/religion/rites/musician_rites.dm]: IMO, this should be checking antimagic, not whether or not you're a priest

maybe we can finally use that "holy" antimagic that bibles give for something other than defending against revenant magic?

**Comment by ATH1909** [comment]
Ah yes, now the crew can finally fall asleep to my masterpiece: A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A A, played on a harmonica.

**Comment by GhostThunder** [comment]
I can imagine it now, war were declared, everybody gearing up, nukies hit the station, meanwhile chaplain is back there going TOOT TOOT FLUTE FLUTE

**Comment by Dennok** [comment]
We need Sect with ability of add RPG to items.

**Comment by necromanceranne** [comment]
Amazing, the chaplain becomes a bard.

**Comment by Shadowflame909** [comment]
@necromanceranne 

Monk Chaplain when

**Comment by davethwave** [comment]
When Orange man stops hating on martial art code. I can see how to link up the favor into experience gain for monk though.

**Comment by Spepbuthewranglesgithub** [comment]
ah yes, playing megalovania can cause physical harm now, not just mental harm.

**Comment by Ratvar** [comment]
I might be blind, but is Lullaby an infinite aoe stun with no indicator of what caused it? If so, why?

**Comment by ExcessiveUseOfCobblestone** [comment]
no.

It is limited to the max duration of the song (- the initial drowsiness incline) without repeats, favor costs, and the fact you have to re tune prior to starting a new song (if you want to add the effects).

The fact it isn't visual is something I want to address, but i'm not sure the best way to convey it.

**Comment by ExcessiveUseOfCobblestone** [comment]
i'm going to work on this later.

**Pull request closed without merging**